### PR TITLE
Tweak dark buttons

### PR DESCRIPTION
**Pull request by DanTup**
I tweaked these again... Still not entirely happy with the "down" state but I think it's an improvement. Feel free to tweak (or suggest tweaks)!

Before:

Screenshot 2019-03-21 at 1 14 48 pm

After:

Screenshot 2019-03-21 at 1 15 10 pm

You can't tell here, but the text shifts slightly as you click them, so it feels better to use than it looks - though the light theme doesn't do that so I'm not entirely sure if we want it (if so, should we do the same there?).

### CODE REVIEW

File: packages/devtools/web/styles_dark.css [hunk 1]
```diff
@@ -68,12 +68,17 @@ nav.menu .menu-item {
 
 /* Tweak how buttons look in the dark theme to not be so dark */
 .btn:not(.btn-primary), .btn:not(.btn-primary)[disabled] {
-    background-image: linear-gradient(-180deg, #3f3f3f 0%, #3f3f3f 90%);
+    border: 1px solid #222222;
```
Comment (jacob314) [inline packages/devtools/web/styles_dark.css]: this is good.

File: packages/devtools/web/styles_dark.css [hunk 2]
```diff
@@ -68,12 +68,17 @@ nav.menu .menu-item {
...
+    border-radius: 3px;
+    background: linear-gradient(#5f5f5f 5%, #535353 10%, #4f4f4f 100%);
+    box-shadow: 1px 1px 1px rgba(0,0,0,0.2);
+    padding: 3px 10px;
 }
-
 .btn.selected:not(.btn-primary), .btn.selected:not(.btn-primary)[disabled] {
-    background-image: linear-gradient(-180deg, #1f1f1f 0%, #1f1f1f 90%);
-    box-shadow: none;
+    color: #cacbcc;
```
Comment (jacob314) [inline packages/devtools/web/styles_dark.css]: This color doesn't visually jump out at me as being selected.
Maybe we should copy the solid color used to indicate that a row is selected in the list of scripts in the debugger view and use that here.
Note that color might also need to change to be more obvious. For example,. consider making it a dark blue or dark orange color to make selection obvious.

Comment (DanTup) [reply packages/devtools/web/styles_dark.css]: How about this? I tried to make it more like the blue in the inspector icons:

Screenshot 2019-03-25 at 2 15 22 pm

```css
.btn.selected:not(.btn-primary), .btn.selected:not(.btn-primary)[disabled] {
    color: #ffffff;
    background: linear-gradient(#1C547C 5%, #104870 90%, #0C446C 100%);
}
```

**Comment by jacob314** [comment]
The blue looks like it clashes a bit but it is definitely noticeable.
I think this is good enough to check in and a designer can help us make it look better.

**Comment by DanTup** [comment]
Yeah, I'm not entirely sold on it, but no matter what I do I think it looks bad. One problem is knowing which of the buttons are the selected/pressed ones, and which are not.

I'll push the above change (it was slightly more than I pasted here) and if you're happy, merge it. Anyone should feel free to tweak if they can improve it, or suggest changes if they want me to try out.

**Comment by DanTup** [comment]
77b679c79dc90a6df048a4d37ff6222ce3d5dd8a is the additional change for the blue buttons. Also removes the moving-text which I wanted to make look like you'd "pressed" the button but looks wonky with the change to make the button "light up" blue instead.

**Comment by jacob314** [comment]
lgtm

**Pull request merged**